docs/docs/cli/[command].paths.js

```diff
@@ -0,0 +1,97 @@
+import cliDataLoader from "./commands.data";
+
+function traverse(command, paths) {
+  paths.push({
+    params: { command: command.link.replace(/\/cli\//, "") },
+    content: content(command),
+  });
+  (command.items ?? []).forEach((subCommand) => {
+    traverse(subCommand, paths);
+  });
+}
+
+function content(command) {
```

Comment:
I wonder if this could have been written in a templating language, Vitepress should be using markdown? It is quite hard to imagine how the final content will look like.

Reply:
I refactored it to use [ejs](https://ejs.co/) but I don't think it reads much better.